instrumentation/rake/lib/opentelemetry/instrumentation/rake/patches/task.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module OpenTelemetry
+  module Instrumentation
+    module Rake
+      module Patches
+        # Module to prepend to Rask::Task for instrumentation
+        module Task
+          def invoke(*args)
+            attributes = { 'rake.task' => name }
+            tracer.in_span('rake.invoke', attributes: attributes) do
```

Comment:
Since this is only one key-value pair how do you feel about inlining this invocation?

```suggestion
            tracer.in_span('rake.invoke', attributes: { 'rake.task' => name }) do
```

Reply:
updated in 9937238